README.md

```
@@ -15,241 +15,158 @@
-
-## 自定义网站
-编辑 `content/pages/*.json` 文件，自由修改站点各个页面的名称、标题和描述：
-
-## ✍️ 写作指南
+### 开发命令
 
-### 默认元信息
-在 `content/meta.json` 中配置默认的文章元信息：
+| 命令 | 描述 |
+|------|------|
+| `pnpm dev` | 启动本地开发服务器 (http://localhost:3000) |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)